test/vast_parser.js

```diff
@@ -156,9 +157,7 @@ describe('VASTParser', function() {
       ].map(item =>
         (item =>
           it(`should not return NaN for \`${item}\``, () =>
-            isNaN(vastParser.parserUtils.parseDuration(item)).should.eql(
-              false
-            )))(item)
+            isNaN(parserUtils.parseDuration(item)).should.eql(false)))(item)
```

Comment:
this line is impossible to follow without indentations. isn't also one too many `item =>`?

Reply:
whoa yep it's totally unreadable 😅 

But let's change it in another PR ok? I'd like to rework all the unit tests we have anyway :)